2021-07-17 19:21:35,989 - INFO - Start of service availability check
2021-07-17 19:21:35,989 - INFO - Start of service availability check
2021-07-17 19:21:35,990 - INFO - Start of service availability check
2021-07-17 19:21:46,165 - INFO - Start of service availability check
2021-07-17 19:21:46,780 - INFO - Start of service availability check
2021-07-17 19:21:47,080 - INFO - Start of service availability check
2021-07-17 19:21:56,820 - INFO - Start of service availability check
2021-07-17 19:21:57,823 - INFO - Start of service availability check
2021-07-17 19:21:58,996 - INFO - Start of service availability check
2021-07-17 19:24:30,313 - INFO - Start of service availability check
2021-07-17 19:24:30,313 - INFO - Start of service availability check
2021-07-17 19:24:30,313 - INFO - Start of service availability check
2021-07-17 19:24:40,490 - INFO - Start of service availability check
2021-07-17 19:24:41,292 - INFO - Start of service availability check
2021-07-17 19:24:43,968 - INFO - Start of service availability check
2021-07-17 19:24:51,118 - INFO - Start of service availability check
2021-07-17 19:24:52,382 - INFO - Start of service availability check
2021-07-17 19:24:56,760 - INFO - Start of service availability check
2021-07-17 19:25:01,750 - INFO - Start of service availability check
2021-07-17 19:25:03,527 - INFO - Start of service availability check
2021-07-17 19:25:09,237 - INFO - Start of service availability check
2021-07-17 19:25:12,394 - INFO - Start of service availability check
2021-07-17 19:36:46,449 - 123145584164864 - job - INFO - Start of service availability check
2021-07-17 19:36:46,449 - 123145589420032 - job - INFO - Start of service availability check
2021-07-17 19:36:46,449 - 123145594675200 - job - INFO - Start of service availability check
2021-07-17 19:36:56,630 - 123145584164864 - job - INFO - Start of service availability check
2021-07-17 19:36:57,129 - 123145594675200 - job - INFO - Start of service availability check
2021-07-17 19:36:57,885 - 123145589420032 - job - INFO - Start of service availability check
2021-07-17 21:06:10,770 - 4448767488 - postgres - INFO - Database connect established
2021-07-17 21:06:22,732 - 4698541568 - postgres - INFO - Database connect established
2021-07-17 21:06:44,568 - 4444626432 - postgres - INFO - Database connect established
2021-07-17 21:07:16,644 - 4659678720 - postgres - INFO - Database connect established
2021-07-17 21:08:01,359 - 4508007936 - postgres - INFO - Database connect established
2021-07-17 21:09:27,573 - 4483296768 - postgres - INFO - Database connect established
2021-07-17 21:09:27,598 - 4483296768 - metrics_saver - INFO - Table ya_ru created successfully or already exists
2021-07-17 21:09:27,611 - 4483296768 - metrics_saver - INFO - Table exante_eu created successfully or already exists
2021-07-17 21:09:27,612 - 4483296768 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 21:10:22,417 - 4652285440 - postgres - INFO - Database connect established
2021-07-17 21:10:22,440 - 4652285440 - metrics_saver - INFO - Table exante_eu created successfully
2021-07-17 21:10:22,449 - 4652285440 - metrics_saver - INFO - Table habr_com created successfully
2021-07-17 21:10:22,450 - 4652285440 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 21:15:03,320 - 4519071232 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 21:15:03,361 - 4519071232 - postgres - INFO - Database connect established
2021-07-17 21:15:03,379 - 4519071232 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 21:15:12,993 - 4635303424 - producer - INFO - Connected to Kafka successfully
2021-07-17 21:15:12,994 - 4635303424 - prod_runner - INFO - Metrics generator script init finished. Number of threads 3. Starting cycle...
2021-07-17 21:52:02,648 - 123145363824640 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 21:52:02,731 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 21:52:13,999 - 123145363824640 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 21:52:14,001 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 21:52:14,072 - 123145407709184 - metrics_saver - ERROR - Insert failed due to column "none" does not exist
LINE 1: ...atency,regex_valid) values(1626547923, 'ru', 0, 60000, None)
                                                                  ^

2021-07-17 22:01:06,132 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:01:06,142 - 123145363824640 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:01:09,063 - 123145369079808 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', OSError(50, 'Network is down'))
2021-07-17 22:10:08,029 - 123145369079808 - http_wrapper - ERROR - Request failed due to HTTPConnectionPool(host='habr.com', port=80): Max retries exceeded with url: /ru/all (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f908796f8d0>: Failed to establish a new connection: [Errno 50] Network is down'))
2021-07-17 22:10:09,267 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:10:19,369 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:19:12,114 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:28:15,110 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:28:25,226 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:37:18,112 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:37:28,455 - 123145363824640 - http_wrapper - ERROR - Request failed due to HTTPSConnectionPool(host='exante.eu', port=443): Read timed out. (read timeout=60)
2021-07-17 22:37:38,550 - 123145363824640 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:37:40,694 - 123145369079808 - http_wrapper - ERROR - Request failed due to HTTPSConnectionPool(host='habr.com', port=443): Read timed out. (read timeout=60)
2021-07-17 22:46:23,461 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:46:33,748 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:49:51,063 - 123145358569472 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:49:54,356 - 123145369079808 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', OSError(50, 'Network is down'))
2021-07-17 22:50:48,992 - 123145363824640 - http_wrapper - ERROR - Request failed due to HTTPSConnectionPool(host='exante.eu', port=443): Read timed out. (read timeout=60)
2021-07-17 22:50:59,096 - 123145363824640 - http_wrapper - ERROR - Request failed due to ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2021-07-17 22:58:26,892 - 4742188544 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 22:58:26,936 - 4742188544 - postgres - INFO - Database connect established
2021-07-17 22:58:26,949 - 4742188544 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 22:58:47,317 - 4618743296 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 22:58:47,358 - 4618743296 - postgres - INFO - Database connect established
2021-07-17 22:58:47,380 - 4618743296 - metrics_saver - INFO - Table ya_ru created successfully
2021-07-17 22:58:47,380 - 4618743296 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 23:16:54,762 - 4724948480 - producer - INFO - Connected to Kafka successfully
2021-07-17 23:16:54,763 - 4724948480 - prod_runner - INFO - Metrics generator script init finished. Number of threads 3. Starting cycle...
2021-07-17 23:16:57,304 - 4492172800 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 23:16:57,357 - 4492172800 - postgres - INFO - Database connect established
2021-07-17 23:16:57,375 - 4492172800 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 23:19:50,288 - 4726943232 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 23:19:50,320 - 4726943232 - postgres - INFO - Database connect established
2021-07-17 23:19:50,332 - 4726943232 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 23:43:27,192 - 4417273344 - producer - ERROR - Exception while connecting Kafka Unrecognized configs: {'api-version': '(0, 10)'}
2021-07-17 23:43:27,194 - 4417273344 - prod_runner - INFO - Metrics generator script init finished. Number of threads 3. Starting cycle...
2021-07-17 23:43:27,374 - 123145501708288 - producer - ERROR - Exception in publishing message 'NoneType' object has no attribute 'send'
2021-07-17 23:43:28,169 - 123145512218624 - producer - ERROR - Exception in publishing message 'NoneType' object has no attribute 'send'
2021-07-17 23:43:28,274 - 123145506963456 - producer - ERROR - Exception in publishing message 'NoneType' object has no attribute 'send'
2021-07-17 23:44:12,538 - 4563533312 - producer - ERROR - Exception while connecting Kafka invalid literal for int() with base 10: '(0, 10)'
2021-07-17 23:44:12,539 - 4563533312 - prod_runner - INFO - Metrics generator script init finished. Number of threads 3. Starting cycle...
2021-07-17 23:44:12,728 - 123145346633728 - producer - ERROR - Exception in publishing message 'NoneType' object has no attribute 'send'
2021-07-17 23:44:13,640 - 123145357144064 - producer - ERROR - Exception in publishing message 'NoneType' object has no attribute 'send'
2021-07-17 23:44:13,860 - 123145351888896 - producer - ERROR - Exception in publishing message 'NoneType' object has no attribute 'send'
2021-07-17 23:45:31,268 - 4532825600 - producer - INFO - Connected to Kafka successfully
2021-07-17 23:45:31,269 - 4532825600 - prod_runner - INFO - Metrics generator script init finished. Number of threads 3. Starting cycle...
2021-07-17 23:45:37,218 - 4573879808 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 23:45:37,271 - 4573879808 - postgres - INFO - Database connect established
2021-07-17 23:45:37,288 - 4573879808 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 23:58:05,005 - 4555943424 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 23:58:05,072 - 4555943424 - postgres - INFO - Database connect established
2021-07-17 23:58:10,300 - 4555943424 - cons_runner - INFO - Saver script init finished. Starting cycle...
2021-07-17 23:58:47,088 - 4683345408 - consumer - INFO - Connected to Kafka successfully.
2021-07-17 23:58:47,136 - 4683345408 - postgres - INFO - Database connect established
2021-07-17 23:58:47,162 - 4683345408 - cons_runner - INFO - Saver script init finished. Starting cycle...
